### PR TITLE
Edit configuration option names

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: docs/en/advanced/Namespace-Management.md [hunk 1]
```diff
@@ -317,20 +315,20 @@ Active sync also tries to avoid syncing when the target directory is heavily use
...
 The system guarantees that we will start syncing a directory if it is "quiet", or it has not been synced for a long period (period longer than the max age).
 
 For example, the following setting 
 
 ```
-alluxio.master.activesync.interval=30secs
-alluxio.master.activesync.maxactivity=100
-alluxio.master.activesync.maxage=5
+alluxio.master.active.sync.interval=30secs
+alluxio.master.ufs.active.sync.max.activities=100
```
Comment (calvinjia) [inline docs/en/advanced/Namespace-Management.md]: Why are some of the configurations prefixed with `ufs` and others not?

Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: oops, fixed. thanks.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**